### DISCUSSION TITLE
finn setup

### DISCUSSION THREAD

**Discussion by harrisonasd** [Xilinx/finn#984] [Q&A] [answered]
During the 'finn setup' stage, how can I inform the system that I intend to use the rfsoc4x2 board?
![螢幕擷取畫面 2024-02-18 165209](https://github.com/Xilinx/finn/assets/105301827/3948face-dacd-4610-9713-05c4a1327777)

![螢幕擷取畫面 2024-02-18 164634](https://github.com/Xilinx/finn/assets/105301827/41e91c42-63fd-465e-8081-a08fc6366c6b)
n/assets/105301827/752b1c1b-d28b-401f-a565-03d515d31e21)

**Comment by fpjentzsch** [comment, marked as answer]
The PYNQ_BOARD env var is used only for certain end-to-end tests. Normally you would use FINN's "build_dataflow" feature to launch the accelerator build. In the builder config you can set the board parameter: https://github.com/Xilinx/finn/blob/b3bdff118ae076cb776af6e51ddc28eeaa0d6390/src/finn/builder/build_dataflow_config.py#L239
